src/Models/HydrostaticFreeSurfaceModels/calculate_hydrostatic_free_surface_tendencies.jl

```diff
@@ -46,34 +46,31 @@ end
 using Oceananigans.TurbulenceClosures.CATKEVerticalDiffusivities: FlavorOfCATKE
 using Oceananigans.TurbulenceClosures.MEWSVerticalDiffusivities: MEWS
 
-const HFSM = HydrostaticFreeSurfaceModel
-
-# Fallback
-@inline tracer_tendency_kernel_function(model::HFSM, name, c, K)                     = hydrostatic_free_surface_tracer_tendency, c, K
-@inline tracer_tendency_kernel_function(model::HFSM, ::Val{:K}, c::MEWS,          K) = hydrostatic_turbulent_kinetic_energy_tendency, c, K
-@inline tracer_tendency_kernel_function(model::HFSM, ::Val{:e}, c::FlavorOfCATKE, K) = hydrostatic_turbulent_kinetic_energy_tendency, c, K
+@inline tracer_tendency_kernel_function(model::HFSM, name, c, K)                     = calculate_hydrostatic_free_surface_Gc!, c, K
+@inline tracer_tendency_kernel_function(model::HFSM, ::Val{:K}, c::MEWS,          K) = calculate_hydrostatic_free_surface_Ge!, c, K
+@inline tracer_tendency_kernel_function(model::HFSM, ::Val{:e}, c::FlavorOfCATKE, K) = calculate_hydrostatic_free_surface_Ge!, c, K
 
 function tracer_tendency_kernel_function(model::HFSM, ::Val{:e}, closures::Tuple, diffusivity_fields::Tuple)
     catke_index = findfirst(c -> c isa FlavorOfCATKE, closures)
 
     if isnothing(catke_index)
-        return hydrostatic_free_surface_tracer_tendency, closures, diffusivity_fields
+        return calculate_hydrostatic_free_surface_Gc!, closures, diffusivity_fields
     else
         catke_closure = closures[catke_index]
         catke_diffusivity_fields = diffusivity_fields[catke_index]
-        return hydrostatic_turbulent_kinetic_energy_tendency, catke_closure, catke_diffusivity_fields 
+        return calculate_hydrostatic_free_surface_Ge!, catke_closure, catke_diffusivity_fields 
     end
 end
 
 function tracer_tendency_kernel_function(model::HFSM, ::Val{:K}, closures::Tuple, diffusivity_fields::Tuple)
     mews_index = findfirst(c -> c isa MEWS, closures)
 
     if isnothing(mews_index)
-        return hydrostatic_free_surface_tracer_tendency, closures, diffusivity_fields
+        return calculate_hydrostatic_free_surface_Gc!, closures, diffusivity_fields
     else
         mews_closure = closures[mews_index]
         mews_diffusivity_fields = diffusivity_fields[mews_index]
-        return  hydrostatic_turbulent_kinetic_energy_tendency, mews_closure, mews_diffusivity_fields 
+        return  calculate_hydrostatic_free_surface_Ge!, mews_closure, mews_diffusivity_fields 
```

Comment:
I like the old name, it's more clear. Can we change it back?

Reply:
this is the kernel, not the function